staging/src/kubevirt.io/api/migrations/v1alpha1/types.go

```diff
@@ -161,7 +161,7 @@ func countMatchingLabels(policy *MigrationPolicy, vmiLabels, namespaceLabels map
 	countLabelsHelper := func(policyLabels, labelsToMatch map[string]string) {
 		for policyKey, policyValue := range policyLabels {
 			value, exists := labelsToMatch[policyKey]
-			if exists && value == policyValue {
+			if exists && (value == policyValue || policyValue == "") {
```

Comment:
A helper function should not be in the api package. Please move this closer to the places where it is moved. Please move all these functions out of the api package.